libs/components/src/wizard/vl-wizard.component.cy.ts

```diff
@@ -44,9 +55,35 @@ describe('component vl-wizard - default', () => {
         cy.get('vl-wizard').find('vl-wizard-pane').should('not.have.length', 4);
     });
 
-    it.only('should render correct step text', () => {
+    it('should render correct <pane.name> when the pane is hovered', () => {
+        mountDefault({
+            panes: ['Step 1', 'Step 2', 'Step 3'],
+            activeStep: 1,
+        });
+
+        cy.get('vl-wizard').find('vl-wizard-pane[data-vl-name="Step 1"]').trigger('mouseover');
+        cy.get('vl-wizard').find('vl-wizard-pane[data-vl-name="Step 1"]').contains('Wizard Pane Content (Step 1)');
+    });
+
+    it('should set correct pane as active', () => {
+        mountDefault({
+            panes: ['Step 1', 'Step 2', 'Step 3'],
+            activeStep: 2,
+        });
+        cy.get('vl-wizard').find('vl-wizard-pane[data-vl-name="Step 2"]').contains('Wizard Pane Content (Step 2)');
+    });
+
+    it('should not display the steps\' labels by default', () => {
         mountDefault({ panes: ['Step 1', 'Step 2', 'Step 3'] });
 
+        cy.get('vl-wizard').should('not.have.attr', 'data-vl-show-steps');
```

Comment:
Afchecken of het attribuut erop staat is op zich niet nodig maar ik zou hier wel zeker controleren in de html dat de labels van de steps ook effectief niet worden getoond. Dus checken dat `'.vl-progress-bar__bullet__text'` niet aanwezig is bijvoorbeeld.

Reply:
gedaan